vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -57,15 +57,15 @@
   <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactCommunity.cpp.props" />
   <ItemDefinitionGroup Condition="'$(OSS_RN)'=='true'">
     <ClCompile>
-      <DisableSpecificWarnings>4244;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings>4244;4267;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
I assigned an internal bug to myself to upstream our fix for this.

Reply:
At some point we should look at cleaning up upstream of a bunch of warnings, so we can turn these all off.